File: API/Dtos/ApprovalComicDto.cs
using API.Helpers;

namespace API.Dtos
{
    public class ApprovalComicDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime CreationTime { get; set; }
        public string MainImage { get; set; }
        public string Desc { get; set; }
        public string Genres { get; set; }
        public ApprovalStatusComic ApprovalStatus { get; set; }
        public int AuthorId { get; set; }
    }
}

File: client/src/app/_models/approvalComicDto.ts
import { ApprovalStatusComic } from '../_extensions/enumHelper';

export interface ApprovalComicDto {
  id: number;
  name: string;
  creationTime: Date;
  approvalStatus: ApprovalStatusComic;
  authorId: number;
  mainImage: string;
  desc: string;
  genres: string;
}
